src/main/scala/cognite/spark/v1/DefaultSource.scala

```diff
@@ -472,6 +475,33 @@ object DefaultSource {
     )
   }
 
+  private[v1] def parseCogniteIds(paramName: String)(
+      parameters: Map[String, String]): Option[List[CogniteId]] = {
```

Comment:
Any specific reason why `paramName` & `parameters` are in 2 parameter lists? 🤔

Reply:
not really 🤔 .... it started as a function receiving the map and with hardcoded key, and then added the `paramName`.... but now that you mention it, probably it'll be easier to read if it receives just the string 😅